src/Microsoft.Data.Analysis/Microsoft.Data.Analysis.csproj

```diff
@@ -5,7 +5,7 @@
     <LangVersion>7.3</LangVersion>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <SuppressFinalPackageVersion>false</SuppressFinalPackageVersion>
-    <VersionPrefix>0.3.0</VersionPrefix>
+    <VersionPrefix>0.4.0</VersionPrefix>
     <IsShipping>true</IsShipping>
```

Comment:
Nit: Now that you're on this we can remove the `IsShipping` property. That is set to true by default in arcade, and we were using this for  the old mechanism to set stable packages before we adopted arcade in this repo.